apps/mobile/src/screens/Tips/index.tsx

```diff
@@ -50,7 +50,7 @@ export const Tips: React.FC = () => {
       {selectedTab !== SelectedTab.MESSAGES && (
         <Pressable
           style={styles.createPostButton}
-          onPress={() => navigation.navigate('MainStack', {screen: 'CreateForm'})}
+          onPress={() => Linking.openURL('https://afk-community.xyz/create-form')}
```

Comment:
Why change this? I'm not sure it's needed (and not cool when you are in dev ahah).
Can you readd the navigation.navigate() one?